### PR TITLE
[FR] Add translations for stubs from the "Gameplay" and "Input device" categories

### PR DESCRIPTION
**Pull request by DoryanR**
Translation of the following wiki pages:

- `Gameplay/Accuracy`
- `Gameplay/Drain_time`
- `Gameplay/Fingerlock`
- `Gameplay/Input_device/Beatmania_controller`
- `Gameplay/Input_device/Dance_pad`
- `Gameplay/Input_device/Drum_controller`
- `Gameplay/Input_device/Graphics_tablet`
- `Gameplay/Input_device/Keyboard`
- `Gameplay/Input_device/Mouse`
- `Gameplay/Input_device/Touch_device`
- `Gameplay/Input_device/Wiimote`
- `Gameplay/Input_device`
- `Gameplay/Mind_block`
- `Gameplay/Play_time`
- `Gameplay/Slider_break`
- `Gameplay/Spectating`
- `Gameplay/Tablet_cover`
- `Gameplay`

Also in [EN] files :

- In `Gameplay/Drain_time` : fix typo

**Notify me if the PR is merge without review, that I self-review before.**

### CODE REVIEW

File: wiki/Gameplay/Accuracy/fr.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Précision
+
+<!-- TODO: needs to be combined with /wiki/Accuracy in some way -->
+
+**La précision** (abrégé en *acc*) définit la précision avec laquelle un joueur frappe les [hits objects](/wiki/Hit_object) sur une [beatmap](/wiki/Beatmap). Cette valeur varie de 0 à 100%. La précision peut être vue dans le coin supérieur droit de l'écran lorsque vous regardez un replay ou jouez à une beatmap.
```
Comment (Pachiru) [inline wiki/Gameplay/Accuracy/fr.md:9]:
```suggestion
**La précision** (abrégé en *acc* pour *accuracy*) définit la précision avec laquelle un joueur frappe les [hits objects](/wiki/Hit_object) sur une [beatmap](/wiki/Beatmap). Cette valeur varie de 0 à 100%. La précision peut être vue dans le coin supérieur droit de l'écran lorsque vous regardez un replay ou jouez à une beatmap.
```

File: wiki/Gameplay/Drain_time/fr.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+stub: true
+tags:
+  - drain length
+---
+
+# Drain time
+
+Le **Drain time** désigne la durée totale d'une [beatmap](/wiki/Beatmap) qui nécessite que le joueur soit actif. Elle commence au premier [hit object](/wiki/Hit_Objects) et ce termine à la fin du dernier hit object et ne comprend pas les pauses. Le mot *drain* provient du système de [osu!](/wiki/Game_mode/osu!) et de [osu!catch](/wiki/Game_mode/osu!catch) de l'[health drain](/wiki/Beatmapping/Health_Drain), où la santé du joueur diminue constamment au fil de la carte.
```
Comment (Pachiru) [inline wiki/Gameplay/Drain_time/fr.md:9]:
```suggestion
Le **Drain time** désigne la durée totale d'une [beatmap](/wiki/Beatmap) durant laquelle le joueur sera actif. Elle commence au premier [hit object](/wiki/Hit_Objects) et ce termine à la fin du dernier hit object et ne comprend pas les pauses. Le mot *drain* provient du système d'[health drain](/wiki/Beatmapping/Health_Drain) intégré dans les modes [osu!](/wiki/Game_mode/osu!) et [osu!catch](/wiki/Game_mode/osu!catch), où la santé du joueur diminue constamment au fil de la carte.
```
Le mot *drain* provient du système de [osu!](/wiki/Game_mode/osu!) et de [osu!catch](/wiki/Game_mode/osu!catch) de l'[health drain](/wiki/Beatmapping/Health_Drain)

File: wiki/Gameplay/Drain_time/fr.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+Les beatmap avec des drain time très longs sont généralement appelés *[marathons](/wiki/Glossary#marathon)* car ils nécessitent une endurance hors norme pour être terminés.
+
+La page d'information de la beatmap sur le site web calcule le drain time (étiqueté `Drain length`) en se basant uniquement sur les heures de début des hits objects, donc si le dernier objet de la carte est un [slider](/wiki/Hit_object/Slider) ou un [spinner](/wiki/Hit_object/Spinner), le drain time sera affiché de manière incorrecte.
```
Comment (Pachiru) [inline wiki/Gameplay/Drain_time/fr.md:13]:
```suggestion
La page d'information de la beatmap sur le site web calcule le drain time (étiqueté `Drain length`) en se basant uniquement sur les début des hits objects, donc si le dernier objet de la beatmap est un [slider](/wiki/Hit_object/Slider) ou un [spinner](/wiki/Hit_object/Spinner), le drain time sera affiché de manière incorrecte.
```

File: wiki/Gameplay/Fingerlock/fr.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Le fingerlock est une sensation courante qui arrive à de nombreux joueurs d'osu! expérimentés. La plupart des joueurs s'accordent à dire qu'il n'y a généralement pas lieu de s'inquiéter. Cependant, si un joueur ressent souvent un fingerlock persistant ou douloureux, il est recommandé qu'il consulte un professionnel de santé, car cela peut être le résultat d'un problème de santé sous-jacent.
+
+Une cause spécifique du fingerlock n'a pas encore été trouvée ni comprise. Cependant, il existe quelques habitudes communes qui peuvent causer ou augmenter les risques de fingerlock : de longues sessions de jeu (en particulier ceux qui réessaient la même [beatmap](/wiki/Beatmap) à plusieurs reprises), une mauvaise posture ou position du poignet, ou une tension inutile sur le poignet ou les doigts pendant de longues périodes. Quoi qu'il en soit, le fingerlock se produit le plus souvent une ou deux fois au cours d'une session de jeu, à la suite d'une simple hésitation ou d'un temps de réaction lent.
```
Comment (Pachiru) [inline wiki/Gameplay/Fingerlock/fr.md:25]:
```suggestion
Les causes spécifiques du fingerlock n'ont pas encore été trouvées ni comprises. Cependant, il existe quelques habitudes communes qui peuvent causer ou augmenter les risques de fingerlock : de longues sessions de jeu (en particulier ceux qui réessaient la même [beatmap](/wiki/Beatmap) à plusieurs reprises), une mauvaise posture ou position du poignet, ou une tension inutile sur le poignet ou les doigts pendant de longues périodes. Quoi qu'il en soit, le fingerlock se produit le plus souvent une ou deux fois au cours d'une session de jeu, à la suite d'une simple hésitation ou d'un temps de réaction lent.
```

**Review by Pachiru — COMMENTED** [review]
Haven't finished looking through the whole PR so consider the review unfinished

**Review by TicClick — APPROVED** [review]

**Pull request merged**